assets/section-main-product.css

```diff
@@ -648,6 +681,40 @@ a.product__text {
     right: 5rem;
     top: 2.2rem;
   }
+  .image_label {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    padding: 4px 8px;
+    gap: 10px;
+    width: 174px;
+    height: 26px;
+    background: rgba(15, 30, 59, 0.03);
+    border: 1px solid rgba(15, 30, 59, 0.2);
+    border-radius: 6px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    position: relative;
+    bottom: 32px;
+    left: 8px;
+    z-index: 2;
+  }
+  
+  #image_type {
+    width: 158px;
+    height: 18px;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 18px;
+    color: rgba(15, 30, 59, 0.5);
```

Comment:
Please look at the rest of this file on how color is mostly used. Notice that CSS variables are used so that the actual color codes can be defined by the theme in the Shopify Admin GUI, and can be easily managed in one place. Please see if you can also use some of the predefined colors. 
Example: `color: rgb(var(--color-button-text));`

PS. If Jordan has defined some colors for this feature that are not yet defined for the theme ask him and make sure he has a damn good reason for this.
PPS. Also applies for other color usages, rgba, etc above and below

Reply:
Color is now used the way it should (and will be used in the future also).